crates/rspack_plugin_javascript/src/visitors/dependency/context_helper.rs

```diff
@@ -145,6 +162,33 @@ fn scan_context_module_concat_call(expr: &CallExpr) -> Option<(String, String)>
   Some((context, reg))
 }
 
+// require(String.raw`./${a}.js`)
+fn scan_context_module_tagged_tpl(tpl: &TaggedTpl) -> Option<(String, String)> {
```

Comment:
since we return `Some(String, String)` in every branch, so the `Option` looks redundant.